cmd/execute.go

```diff
@@ -49,8 +49,8 @@ type (
 )
 
 var (
-	metricsPort      int
-	metricsAddress   string
```

Comment:
should probably keep and deprecate these, if set, convert them to the new settings and print a warning they are deprectated. Also simplifies rolling out this breaking change.

Reply:
Good idea. Added a crude parser that automatically turns off API level metrics